From 727f884a5f74f8e0a8d184a85af3429f87e84397 Mon Sep 17 00:00:00 2001
From: Koopa <codingkoopa@gmail.com>
Date: Fri, 14 Aug 2020 19:56:38 -0400
Subject: [PATCH] Translate Sandgem town text.

---
 .../convert/message/src/scenario/t02.gmm      | 32 ++++++-------------
 1 file changed, 10 insertions(+), 22 deletions(-)

diff --git a/pm_dp_ose/convert/message/src/scenario/t02.gmm b/pm_dp_ose/convert/message/src/scenario/t02.gmm
index 5d59e7274..2a8b9051c 100755
--- a/pm_dp_ose/convert/message/src/scenario/t02.gmm
+++ b/pm_dp_ose/convert/message/src/scenario/t02.gmm
@@ -271,8 +271,7 @@ posイベント
 
 0＝主人公名前</comment>
 		<attribute name="キャラ指定">ポケモン博士</attribute>
-		<language name="日本語" width="216">はかせ『ウォッホン！
-[1:03:トレーナー名:0]　ついて　きなさい</language>
+		<language name="日本語" width="216">Follow me, Lucas!</language>
 	</row>
 
 	<row id="msg_t02_heroine_01">
@@ -288,19 +287,9 @@ posイベント
 1＝サポート♀名前
 No453「たんけんセット」入手</comment>
 		<attribute name="キャラ指定">サポート♀</attribute>
-		<language name="日本語" width="216">あなた　どうして
-ポケモンずかん　を？▼
-……　……　……
-……　……　……▼
-[1:03:トレーナー名:0]さん……
-いい　なまえ　ですよね▼
-あなたも　はかせの　けんきゅうを
-おてつだいを　なさるんですね▽
-じゃあ　あたしと　おなじ　ですね！▼
-あたし　[1:03:トレーナー名:1]です！
-はかせの　おてつだいを　しています▼
-あっ　そうだ！
-よかったら　これを　どうぞ！</language>
+		<language name="日本語" width="216">Hiya! You're Lucas, right? I'm just like
+you, I'm also completing the Pokédex.▼
+I'm Dawn! Oh, I have something for you.</language>
 	</row>
 
 	<row id="msg_t02_heroine_02">
@@ -314,13 +303,12 @@ No453「たんけんセット」入手</comment>
 0＝サポート♀名前
 1＝主人公名前</comment>
 		<attribute name="キャラ指定">サポート♀</attribute>
-		<language name="日本語" width="216">[1:03:トレーナー名:0]『いま　おわたしした
-たんけんセットを　つかうと▽
-ちかに　いけるように　なるんです！▼
-ぜひ　つかって　くださいね
-いろんな　どうぐも　みつかりますし▼
-それでは　[1:03:トレーナー名:1]さん
-これから　よろしく　おねがいします！</language>
+		<language name="日本語" width="216">If you use this explorer's kit, you'll be
+able to go underground! You'll be able to▼
+find all sorts of neat things down there.
+I look forward to working with you in the▼
+future, Lucas!
+</language>
 	</row>
 
 	<row id="msg_t02_hero_01">
-- 
2.28.0

